darwin/dataset/remote_dataset.py

```diff
@@ -249,7 +253,7 @@ def pull(
 
         if release.latest:
             latest_dir = self.local_releases_path / "latest"
-            if latest_dir.exists():
+            if os.path.islink(latest_dir):
```

Comment:
I think we can leverage the `pathlib` equivalent
```suggestion
            if latest_dir.is_symlink():
```
We can avoid importing `os` that way

Reply:
Pathlib `is_symlink()` breaks the code on my Windows 10 machine. Can you verify if it works for you?

Reply:
Done.